tests/x11/turn_off_gnome_screensaver.pm

```diff
@@ -0,0 +1,21 @@
+# SUSE's openQA tests
+#
+# Copyright 2020-2021 SUSE LLC
```

Comment:
```suggestion
# Copyright SUSE LLC
```

Reply:
Done.

Reply:
Not done. Please read the contribution hints in this project root folder and see my *specific* suggestion

Reply:
But better first see my review comment about the test module itself

Reply:
The file is deleted.